daemon/data_manager.go

```diff
@@ -219,9 +219,13 @@ func toServerTechnology(
 			} else {
 				serverTechnology = core.OpenVPNUDP
 			}
+		case config.Protocol_Webtunnel:
```

Comment:
Is it needed here this case to check that irrelevant protocol is not supported by this technology?

Reply:
Ok, I get it - to satisfy linter